sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +303,24 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def get_2d_plot_limts(self):
+        """
+        Update vmin/vmax when set to 'auto'.
+        """
+        vmin, vmax = None, None
+        if self.imshow_kwargs.get('vmax') == 'auto':
+            vmin, vmax = AsymmetricPercentileInterval(*((1, 90)*u.percent).to('%').value)\
+                         .get_limits(self.data_transposed)
```

Comment:
```suggestion
            percent_limits = (1, 90)*u.percent).to('%').value
            return AsymmetricPercentileInterval(percent_limits).get_limits(self.data_transposed)
```
Would this not work?

Reply:
This would but when ```vmax``` isn't explicitly specified, we would have to specify a separate else and return [None, None]. It looked neater having to return ```vmin, vmax```

Reply:
I updated it to account for that.